utils/dataconvert/dataconvert.cpp

```diff
@@ -1574,6 +1575,44 @@ boost::any DataConvert::StringToTimestamp(const datatypes::ConvertFromStringPara
   return value;
 }
 
+//------------------------------------------------------------------------------
```

Comment:
The code from dataconvert must be separated and wrapped into datatypes TypeHandler classes in datatypes/mcs_datatype.*

Reply:
All the changes from dataconvert.cpp must end up in the above mentioned data type files.